config/composer-35.json

```diff
@@ -37,7 +37,7 @@
     "symfony/process": ">=3.4.47 <7.0",
     "symfony/filesystem": ">=3.4.47 <7.0",
     "vlucas/phpdotenv": "^4.3",
-    "ifsnop/mysqldump-php": "^2.12"
+    "druidfi/mysqldump-php": "^1.1"
```

Comment:
this dependency is requiring PHP 7.4+

is it fine to raise the min php version?

Reply:
Version 3.5 must be compatible with PHP 7.1 to 7.4, for this reason, the dependency cannot be changed.
It's fine to update `master` with the new solution, but 3.5 will have to keep the existing one.

Reply:
thanks for pushing it over the finishing line